src/lesson3/Java_Beginner_chapter_1/Task16.java

```diff
@@ -0,0 +1,44 @@
+package lesson3.Java_Beginner_chapter_1;
+
+import java.util.Random;
+import java.util.Scanner;
+
+//Определите сумму элементов одномерного массива, расположенных между
+//минимальным и максимальным значениями.
```

Comment:
Print sum between min and max
array:
1 3 6 3 8 4
min = 1
max = 8 
sum = 3 + 6 + 3 = 12

Reply:
I fixed